.travis.yml

```diff
@@ -35,11 +29,29 @@ addons:
     - dejagnu
     - jq
     - opam
+
+cache:
+  directories:
+  - $HOME/.opam
+  - $HOME/save_opam
```

Comment:
why do we need to copy from `save_opam` to `.opam` if they both are cached?

Reply:
We can environment variables and add `save_opam` to path lookup. Also, we can change tests in `bap-testsuite`, e.g. for `run` tests, we will have to add where is a `site-lisp` directory and so on. But I don't think it's a good solution.

Reply:
Ah, better to answer that `bap` is removed while caching of `.opam`, because 
travis does rather a straightforward sequence of actions: install the package and then remove the package. So the only place we can catch this - by setting `POST_INSTALL_HOOK` variable, to save all we can

Reply:
yeah, the last answer clarifies it to me.